.github/workflows/publish-docs.yml

```diff
@@ -21,26 +21,31 @@ jobs:
         with:
           python-version: "3.12"
 
+      - name: Install uv
+        uses: astral-sh/setup-uv@v5
+        with:
+          enable-cache: true
+          cache-dependency-glob: "uv.lock"
+
       - name: Create and activate virtual environment
         run: |
-          python -m venv venv
+          uv venv venv
           source venv/bin/activate
 
       - name: Install pdoc and other dependencies
         run: |
           source venv/bin/activate
-          python -m pip install pdoc3 setuptools
+          uv install pdoc3 setuptools
 
       - name: Install package
         run: |
           source venv/bin/activate
-          pip install ./livekit-agents
-          ./livekit-plugins/install_local.sh
+          uv pip install ./livekit-agents
```

Comment:
i think we might want `uv sync`, because we'd want all the plugins to be installed too in order to generate their docs

Reply:
Right :)